### ISSUE TITLE
Add Sentry to examples iframe

### ISSUE THREAD

**Issue by katydecorah** [mapbox/mapbox-gl-js-docs#271]
On each example page, the interactive map is loaded in a dynamic iframe. As a result, the Sentry project we initialize to catch errors on the docs subdomain is unable to track errors that may occur in the iframe. 

I experimented yesterday and found that within the iframe, we can load Sentry via the CDN  https://docs.sentry.io/platforms/javascript/ and then initialize a Sentry project to catch errors:

- Test commit: https://github.com/mapbox/mapbox-gl-js-docs/commit/2f50ebfb9beb72834e679ff9284612b430185e77 
- The previous test commit generated this Sentry issue: https://sentry.io/organizations/mapboxcom/issues/1592035270/

I think this data could be very useful in not only making sure our examples are working well, but also, providing logs for different environments for the gl js team.

The one thing I noticed about the test Sentry issue above is that the line it's pointing to in the source code isn't correct. Because Sentry relies on source maps to build the source code preview and this content is in a dynamic iframe, it's unlikely that the source will point to an actual line from the iframe's code. I'm not sure if this is a problem in the long run. We will still receive errors and all other metadata associated with the page and the user's environment, but we won't know exactly which line in the iframe kicked off that error.

If we'd like to move forward, I think we should set up a separate Sentry project so that it only has errors from these GL JS examples.

@asheemmamoowala @danswick let me know if you'd like to move forward. The overall implementation should be quick.

**Comment by asheemmamoowala** [comment]
> I think this data could be very useful in [..] providing logs for different environments for the gl js team.

👍 @katydecorah Thanks for setting up the test to show what this would look like. 

> I'm not sure if this is a problem in the long run. We will still receive errors and all other metadata associated with the page and the user's environment, but we won't know exactly which line in the iframe kicked off that error.

Investigating the dynamic iFrame in Chrome devtools I see that it is able to load the gl-js source map posted to the CDN. If sentry can grab that, it would be good enough catch the errors where they are being thrown from GL-JS. The examples are usually quite simple, so it should be fairly easy to work through an error without the call stack that originates the exception.

cc @mapbox/gl-js

**Comment by katydecorah** [comment]
I created a new Sentry project [`gl-js-examples`](https://sentry.io/organizations/mapboxcom/issues/?project=5191235) that will catch any issues found in an example iframe. (Mapbox team members can log into Sentry through Okta.)

* 🎉 I set this Sentry project up as Browser JavaScript (rather than React) and it is now pointing to the correct line in the code where the error was thrown. [Example issue.](https://sentry.io/organizations/mapboxcom/issues/1600200639/)
* The work to add Sentry to the iframe is happening in https://github.com/mapbox/mapbox-gl-js-docs/pull/274
* For monitoring, I created an [alert](https://sentry.io/settings/mapboxcom/projects/gl-js-examples/alerts/): when 5 users see the same issue in one week or an issue goes from resolved to unresolved (regression) then the issue will get posted to the doc's team Sentry channel. This threshold has worked well for the docs-wide Sentry project. The docs team will create GitHub issues as needed.
